### ISSUE TITLE
More than one account connected from a computer

### ISSUE THREAD

**Issue by Porkepix** [Manuel-777/MTG-Arena-Tool#654]
This issue is equally an issue and questions.
Today I lended/loaned (no idea which one fits the best here) my computer to someone else to play on it. Which means another account connected and writing in MTGA logs.

The immediate result was to see its account on MTG Arena Tool (which btw raise in itself privacy issues imho: I could see its decks and collection, gold, gems and so on, as well as its game activity on my computer).
I had no access anymore to my own data, until I connect again my own account in MTGA.

Now, issues and questions:
- Is it possible to have a clean way to see accounts different from the last connected one?
- Is it normal to have access to someone else's data with situations like the described scenario?
- As MTG Arena Tool keeps data by itself in its own database (we talk here about local database, no account), what happens to data from another account than the last connected, visible in MTG Arena Tool?
- Now, if I had a MTG Arena Tool account, would this guest's MTGA account be added to mine? What if he also had a MTG Arena Tool?

**Comment by Manuel-777** [comment]
In that scenario MTG Arena Tool would close your config file (that contains all of your history, matches and data) and open a new one for the user that was loggen in. and load any previous data if there is.
We are still exploring and fixing issues that may arise in this situation, but the idea is that, despite you using your mtgatool account with multiple arena accounts, the data *should not* be merged at any point. This includes economy transactions, matches, events, etc.

It is normal to have this scenario, and the ideal here is to have a single mtgatool account for a single computer (to avoind having to log in and out in case someone else decides to play). And have all data separated by arena user ID. It would not affect winrates and such.

As I said we are still exploring the issues and solutions to this, since I dont usually switch accounts it might be troublesome to test deeply and some things might be working oddly/not as I described. Please send any incorrect behaviour and the steps required to reproduce them!

**Comment by Porkepix** [comment]
To clarify, part of my previous message are theoritical question: the use case I had was to lend/loan my computer to someone else, MTG Arene Tool was not running at that time, and I only used it for local/offline accounts. No MTG Arena Tool accounts involved.

> In that scenario MTG Arena Tool would close your config file (that contains all of your history, matches and data) and open a new one for the user that was loggen in. and load any previous data if there is.

If I understand this correctly, I can assume that despite this guest's MTGA account disconnected and not directly visible anymore, all of its data are still here, on my computer, but not accessible via GUI in the application? So whether it is to read or to clean it, it's needed to fiddle with database files?

**Comment by Manuel-777** [comment]
yes, and to be clear, the only data that is persisted in configuration files from arena tool are settings, matches/rank/events history, economy events and drafts.

**Comment by AnnanFay** [comment]
Related #581
